nupic/research/frameworks/pytorch/restore_utils.py

```diff
@@ -224,6 +231,60 @@ def get_state_dict(checkpoint_path):
         return None
 
 
+def get_module_attr(module, name):
```

Comment:
`get_module_attr` and `set_module_attr` are generic functions that could be used outside the checkpoint restoring context, maybe these function could be moved to `model_utils.py` instead

Reply:
Made this change